.travis.yml

```diff
@@ -77,3 +78,4 @@ install:
 script:
   - industrial_ci/_wrap_test.sh # this script is only for internal use
   - if ! [ -z "$POST_PROCESS" ]; then $POST_PROCESS; fi
+  - if ! [ -z "$DOCKER_COMMIT" ]; then  echo "Test if the Docker image is committed or not. See the size of the Docker image is non-zero or not."; if docker image inspect $DOCKER_COMMIT --format='{{.Size}}'; then exit 0; else exit 1; fi fi
```

Comment:
No need for the inner if branch

Reply:
I'm not sure what you exactly mean. Does https://github.com/AustinDeric/industrial_ci/pull/3 look better @ipa-mdl?